PowerEditor/installer/nativeLang/dutch.xml

```diff
@@ -92,403 +93,416 @@
+					<Item id="43027" name="Stijl 3 wissen"/>
+					<Item id="43028" name="Met behulp van stijl 4"/>
+					<Item id="43029" name="Stijl 4 wissen"/>
+					<Item id="43030" name="Met behulp van stijl 5"/>
+					<Item id="43031" name="Stijl 5 wissen"/>
+					<Item id="43032" name="Alle stijlen wissen"/>
+					<Item id="43033" name="Stijl 1"/>
+					<Item id="43034" name="Stijl 2"/>
+					<Item id="43035" name="Stijl 3"/>
+					<Item id="43036" name="Stijl 4"/>
+					<Item id="43037" name="Stijl 5"/>
+					<Item id="43038" name="Stijl zoeken"/>
+					<Item id="43039" name="Stijl 1"/>
+					<Item id="43040" name="Stijl 2"/>
+					<Item id="43041" name="Stijl 3"/>
+					<Item id="43042" name="Stijl 4"/>
+					<Item id="43043" name="Stijl 5"/>
+					<Item id="43044" name="Stijl zoeken"/>
+					<Item id="43055" name="Stijl 1"/>
+					<Item id="43056" name="Stijl 2"/>
+					<Item id="43057" name="Stijl 3"/>
+					<Item id="43058" name="Stijl 4"/>
+					<Item id="43059" name="Stijl 5"/>
+					<Item id="43060" name="Alle stijlen"/>
+					<Item id="43061" name="Stijl zoeken (gemarkeerd)"/>
 					<Item id="43045" name="Gevonden resultaten"/>
-					<Item id="43046" name="Volgende zoekterm"/>
-					<Item id="43047" name="Vorige zoekterm"/>
-					<Item id="43048" name="Markeren en volgende zoeken"/>
-					<Item id="43049" name="Markeren en vorige zoeken"/>
+					<Item id="43046" name="Volgend zoekresultaat"/>
+					<Item id="43047" name="Vorig zoekresultaat"/>
+					<Item id="43048" name="Selecteren en volgende zoeken"/>
+					<Item id="43049" name="Selecteren en vorige zoeken"/>
 					<Item id="43054" name="Markeren..."/>
 					<Item id="44009" name="Interface verbergen"/>
 					<Item id="44010" name="Gegevensstructuur samenvouwen"/>
-					<Item id="44019" name="Alle karakters weergeven"/>
-					<Item id="44020" name="Insprong weergeven"/>
-					<Item id="44022" name="Automatische &amp;terugloop"/>
-					<Item id="44023" name="Inzoomen"/>
-					<Item id="44024" name="Uitzoomen"/>
+					<Item id="44019" name="Alle tekens weergeven"/>
+					<Item id="44020" name="Aanduiding voor inspringing weergeven"/>
+					<Item id="44022" name="Automatische terugloop"/>
+					<Item id="44023" name="&amp;Inzoomen (Ctrl + Omhoog scrollen)"/>
+					<Item id="44024" name="&amp;Uitzoomen (Ctrl + Omlaag scrollen)"/>
 					<Item id="44025" name="Spaties en tabs weergeven"/>
 					<Item id="44026" name="Regeleindemarkering weergeven"/>
 					<Item id="44029" name="Gegevensstructuur uitvouwen"/>
-					<Item id="44030" name="Huidige sectie samenvouwen"/>
-					<Item id="44031" name="Huidige sectie uitvouwen"/>
+					<Item id="44030" name="Huidig niveau samenvouwen"/>
+					<Item id="44031" name="Huidig niveau uitvouwen"/>
 					<Item id="44049" name="Samenvatting..."/>
-					<Item id="44080" name="Documentstructuur"/>
+					<Item id="44080" name="Overzichtskaart van document"/>
 					<Item id="44084" name="Functielijst"/>
-					<Item id="44085" name="Map als &amp;werkomgeving"/>
```

Comment:
Any reason to remove `&amp;`

Reply:
@donho English source doesn't contain `&amp;`:

`<Item id="44085" name="Folder as Workspace"/>`

Reply:
OK I see. However, your PR contains too much reformatting.
The PR will be accepted without any reformatting.